src/main/java/frc/robot/Constants.java

```diff
@@ -405,5 +409,25 @@ public static HashMap<Double, Double> getShooterMap() { // TODO: Find this
 
             return map;
         }
+
+        // NOTE - This should be monotonically increasing
+        // Key - Distance in meters
+        // Value - Time in seconds
+        public static HashMap<Double, Double> timeToGoalMap() { // TODO: Find this
```

Comment:
This is not what I had in mind initially, but is possibly a better idea since it is way easier to edit - nice!